docs/src/main/sphinx/installation/cli.rst

```diff
@@ -80,6 +80,14 @@ The detailed behavior is as follows:
   authentication system configuration.
 * Expired tokens force you to log in again.
 
+.. _cli-jwt-auth:
+
+JWT authentication
+^^^^^^^^^^^^^^^^^^
+
+To pass a :doc:`JWT </security/jwt>` to the server, use the
```

Comment:
Turn the sentence around and maybe detail what <token> is (path to filename?)

Use the --access-token.. to pass

Reply:
Reversed the sentence, but this is a bit of a black box. You're supposed to pass the actual Base64-encoded JWT, but where does the user get that?